README.md

```diff
@@ -56,16 +57,24 @@ client.version # Hash {"Version" => ..., "Commit" => ..., ...}
 
 ## [The `cat` command](#cat)
 
-_Implemented. Need to be documented_
+```ruby
+client.cat('multihash').to_s # => "data\n"
```

Comment:
can you use real multihash?

It's important for an example to actually works an be reproductible.
As a data hashed will always output the same hash it's easy to make an explicit example using real data.

I think of something like:

```
$ echo 'some funny stuff' | ipfs add
Qmblahblahblah

irb > ipfs.cat('Qmblahblahblah').to_s
=> "some funny stuff"
```

obviously my point is that `Qmblahblahblah` would be a real hash... :tired:.